# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../esdl/gcc-4.0.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- esdl-0.94.1025_orig/c_src/esdl_video.c	2004-10-25 11:27:07.000000000 +0200
+++ esdl-0.94.1025/c_src/esdl_video.c	2005-09-09 17:37:55.873252536 +0200
@@ -75,9 +75,9 @@
 {
    char *bp, *start;
    int sendlen;
-   SDL_VideoInfo *vi;
+   const SDL_VideoInfo *vi;
    bp = start = sdl_get_temp_buff(sd, 9+2*4);
-   (const SDL_VideoInfo *) vi = SDL_GetVideoInfo();
+   vi = SDL_GetVideoInfo();
    if(vi->hw_available)   put8(bp, 1);
    else                   put8(bp, 0);
    if(vi->wm_available)   put8(bp, 1);
